src/HttpCache/CacheTagsFormattingPurgerInterface.php

```diff
@@ -0,0 +1,31 @@
+<?php
+
+/*
+ * This file is part of the API Platform project.
+ *
+ * (c) Kévin Dunglas <dunglas@gmail.com>
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+declare(strict_types=1);
+
+namespace ApiPlatform\Core\HttpCache;
+
+/**
+ * Allows cache purgers to format the cache tags header according
+ * to their own needs.
+ *
+ * @author Yanick Witschi <yanick.witschi@terminal42.ch>
+ */
+interface CacheTagsFormattingPurgerInterface extends PurgerInterface
```

Comment:
WDYT about not extending `PurgerInterface` and renaming it `CacheTagsFormatterInterface`? It will allow to extract this logic in a separated class.

Reply:
Yeah, thought about that too but then again, where would you put that? It should not reside in the `HttpCache` namespace etc. so I thought I'm going to create a very specific interface for that purpose only.

Reply:
Why not in `HttpCache`? This namespace looks good to me.